### DISCUSSION TITLE
Daily 27/9

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#36] [General]
Ayer continué practicando Python, hoy estuve en la clase de inglés y durante el día voy a hacer un update del IEEE. También reporté que no podemos acceder a los grupos y voy a proponer un mapa de sitio que lo voy a incluir en una nueva discusión para que tengamos ya preparado el alcance máximo del sistema. También quiero ahora sí agregar más tareas de usuario.

**Comment by NataliaAlvarezIspc** [comment]
Ayer avancé muy poco con  Python y con las historias de usuarios.
Hoy estaré con esos dos temas y practicando CSS. 
Python me está costando mucho y se me hace, a veces, incomprensible

**Reply by rpgrca** [reply]
Si tenés dudas armá una discusión con lo de Python o armamos una llamada como para resolver dudas.

**Reply by NataliaAlvarezIspc** [reply]
Gracias Roberto! Creo que me falta práctica y tiempo para practicar. Voy a intentar meterme con los ejercicios de la plataforma de Ispc y cualquier cosa aviso.

**Comment by AhumadaCeleste** [comment]
Ayer practique HTML. Estoy con dificultades de tiempo lo que no me permite avanzar como quisiera en la semana y sobre cargándome los fines de semana. Espero poder retomar Python en cuanto pase el examen de FullSatck.

**Comment by Thiagox749** [comment]
Estos dias he estado con problemas personales, hoy ya me puse al dia dentro de todo. Mañana realizare el examen de ingles, y retomare python para poder finalizar el tercer modulo.

**Comment by rocappellano** [comment]
Estudié inglés, para mañana realizar el examen, no pude adelantar más debido a trabajos atrasados de otra carrera

**Comment by claubott** [comment]
hoy resolví la actividad de SGC y continué un poco con POO
